src/main/java/fr/inria/diversify/automaticbuilder/GradleAutomaticBuilder.java

```diff
@@ -167,4 +233,57 @@ private String getPrintClasspathTask() {
                 "}";
     }
 
+    private String getPitTask() {
+        return getPitTask(null);
+    }
+
+
+    private String getPitTask(CtType<?> testClass) {
+        return getPitTaskConfiguration() + getPitTaskOptions(testClass);
+    }
+
+    private String getPitTaskConfiguration() {
+        return NEW_LINE + NEW_LINE + "buildscript {" + NEW_LINE +
+                "    repositories {" + NEW_LINE +
```

Comment:
I do not know gradle very well. But if the build.gradle contains already a `buildscript` section, it is ok to add a new one at the end? There is no risk to clash?